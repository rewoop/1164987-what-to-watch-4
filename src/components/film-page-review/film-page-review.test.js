import React from "react";
import renderer from "react-test-renderer";
import FilmPageReview from "./film-page-review";

const Settings = {
  id: `0`,
  author: `Kate Muir`,
  text: `Discerning travellers and Wes Anderson fans will luxuriate in the glorious Mittel-European kitsch of one of the director's funniest and most exquisitely designed movies in years.`,
  date: `December 24, 2016`,
  rating: `8,9`,
};

it(`Should FilmPageReview render correctly`, () => {
  const tree = renderer
    .create(<FilmPageReview
      review={Settings}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});
