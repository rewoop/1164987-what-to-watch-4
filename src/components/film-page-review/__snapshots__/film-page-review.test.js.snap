// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FilmPageReview render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
      style={
        Object {
          "overflow": "hidden",
          "textOverflow": "ellipsis",
        }
      }
    >
      lalala
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Valera
      </cite>
      <time
        className="review__date"
        dateTime="1996-07-12"
      >
        July 12, 1996
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    5
  </div>
</div>
`;
